app/views/triggeredMigration/CheckHmrcRecordsView.scala.html

```diff
@@ -192,7 +222,11 @@ <h2 class="govuk-summary-card__title" id="foreign-property-heading">
     @soleTraderSection
     @h3(messages("triggered-migration.check-hmrc-records.property.heading"), classes = "govuk-heading-m govuk-!-margin-top-5", optId = Some("property-heading"))
     @propertyBusinessSection
+    @hr()
 
+    @if(viewModel.numberOfCeasedBusinesses != 0) {
```

Comment:
A little nitpicky but it would be better to have it be > 0 otherwise you could technically have -1 ceased businesses and itll show

Reply:
weird, when I did that first it wanted to do a new import but now it seems to work - addressed in the next push